app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -54,7 +54,8 @@ def index
       options[:filters] << {:term => {:composite => false}} if params[:noncomposite]
       options[:filters] << {:term => {:name => params[:name]}} if params[:name]
 
-      respond(:collection => item_search(ContentView, params, options))
+      respond(:collection => item_search(ContentView.includes(:repository => [:product, :environment, :gpg_key]),
```

Comment:
This should be `:repositories` and this list isn't complete.  See https://github.com/Katello/katello/pull/5047